packages/core/src/utils/with-focus.scss

```diff
@@ -1,4 +1,4 @@
-@use '../core' as castor;
+@use '../helpers' as castor;
```

Comment:
Maybe better to use this a `helpers` (without aliasing to `castor`)?

Reply:
Fair point.

Reply:
My concern here is that previously `core -> castor` meant both helpers and other stuff e.g. tokens, themes. Now it's only helpers but still `castor`.

Now for the change, it then sort of makes sense - as users do import the whole Castor, but here internally it's only importing helpers.

Reply:
Fair enough. Will change.